src/main/java/org/jabref/model/database/BibDatabases.java

```diff
@@ -13,19 +14,19 @@ private BibDatabases() {
     }
 
     /**
-     * Gets a collection of bibentries and sets an ID for every entry. After that
+     * Gets a collection of bibEntries and sets an ID for every entry. After that
      * all entries will be inserted into a new BibDatabase.
      *
-     * @param bibentries a collection that contains {@link BibEntry}
+     * @param bibEntries a collection that contains {@link BibEntry}
      * @return BibDatabase that contains the entries
      */
-    public static BibDatabase createDatabase(Collection<BibEntry> bibentries) {
+    public static BibDatabase createDatabase(Collection<BibEntry> bibEntries) {
```

Comment:
I think `entries` as a name is already sufficient.

Reply:
I'm curious why you don't think the clone is necessary. It is my understanding that the `AuxParserResult` should have a separate database with separate `BibEntry`s, and you can't have the same entry objects in the same database, so you have to clone them.

Reply:
The purpose of the clone method is too create a copy with the same data in such a way that you can modify the clone without modifying the original entry. For example for copy/pasting, you do not want to have a literal copy of the same entry as then modification to the pasted entry would transfer to the copied one. 

I thought the AuxParser creates a bunch of new entries, and then the clone is not necessary. But you are right the clone might be necessary because the entries are not created but only copied. My bad! Thanks for pointing it out.